scripts/__SnitchInitialize/__SnitchInitialize.gml

```diff
@@ -65,7 +65,9 @@ function __SnitchInitialize()
         __GAPlatform  = "";
         __GAOSVersion = "";
         __GAGameMakerVersion = "gamemaker " + string_delete(string_delete(GM_runtime_version, 5, 1), 1, 3);
-    
+		
+		//Breadcrumbs
```

Comment:
Spaces not tabs in the future please (I'll accept w/o this change and I can fix it later)